zeroeq/http/requestHandler.cpp

```diff
@@ -139,7 +139,7 @@ struct ConnectionHandler : std::enable_shared_from_this<ConnectionHandler>
         }
 
         _body.append(&range[0], size);
-        _size -= size;
+        _size -= static_cast<int>(size);
```

Comment:
I would change _size to be of type size_t instead of casting. You may need to change _getContentLength to return size_t and add error checking in line 30 to return 0 if the conversion yields a negative number.

Reply:
Cast of _size removed.
The type of the variable _size changed to size_t.
Function _getContentLength now returns a size_t.
Function _getContentLength checks the returned value to avoid negative values.